src/main/kotlin/nl/info/client/brp/BrpClientService.kt

```diff
@@ -28,17 +28,19 @@ import nl.info.zac.util.NoArgConstructor
 import org.eclipse.microprofile.config.inject.ConfigProperty
 import org.eclipse.microprofile.rest.client.inject.RestClient
 import java.util.Optional
+import kotlin.jvm.optionals.getOrNull
 
 @ApplicationScoped
 @AllOpen
 @NoArgConstructor
 class BrpClientService @Inject constructor(
     @RestClient val personenApi: PersonenApi,
+
     @ConfigProperty(name = ENV_VAR_BRP_DOELBINDING_ZOEKMET)
-    private val queryPersonenDefaultPurpose: Optional<String>,
+    private val queryPersonenPurpose: Optional<String>,
```

Comment:
just had this renamed to be more explicit, I think
```
    private val queryPersonenDefaultPurpose: Optional<String>,
```
better describes how this value will (in a next pr) be used in the code. This value is a fall-back for when nothing is supplied by the caller of the `queryPersonen` method...

Unless we will remove this completely, then why bother with this value in the first place? Then we should make this a local constant - super simple !!

Reply:
Until the next PR is there we need these variables. They'll allow us to get rid of the proxy solution.